packages/service-worker/worker/src/driver.ts

```diff
@@ -684,7 +684,7 @@ export class Driver implements Debuggable, UpdateSource {
   /**
    * Retrieve a copy of the latest manifest from the server.
    * Return `null` if `ignoreOfflineError` is true (default: false) and the server or client are
-   * offline (detected as response status 504).
+   * offline (detected as response status 504 or 503).
```

Comment:
I think it would be helpful to include the status test in parenthesis:

```suggestion
   * offline (detected as response status 503 (service unavailable) or 504 (gateway timeout)).
```